### PR TITLE
feat: add GetStockAtTick to database

### PR DESCRIPTION
**Pull request by explodingcamera**
Closing #33

### CODE REVIEW

**Comment by suwandre** [comment]
seems good to me but u can maybe use stock symbol instead of stock id, tests would also be nice

**Comment by Alyks1** [comment]
Yea, as Juandogels said, you should use the Symbol instead of the ID in the query

**Comment by explodingcamera** [comment]
@juandogels we could maybe write some tests together tomorrow. The function now supports both the ID and symbols.

**Comment by suwandre** [comment]
alright im down

File: database/database-stocks_test.go [hunk 1]
```diff
@@ -275,3 +275,77 @@ func (suite *DatabaseTestSuite) TestGetStockMapAtTick() {
...
+			return
+		}
+
+		stockName := test.stockName
+		if stockName == "insert-uuid" {
+			stockName = test.stock.StockID.String()
+		}
+
+		result, err := suite.dbService.GetStockAtTick(stockName, test.tick)
+		if test.needsToFail && assert.Error(err, "didnt't return the wrong tick") {
```
Comment (Alyks1) [inline database/database-stocks_test.go]: The error msg is quite hard to understand - should elaborate

**Review by Alyks1 — CHANGES_REQUESTED** [review]
Generally looks good. I flagged something unclear

**Review by Alyks1 — APPROVED** [review]

**Pull request merged**